.github/workflows/ci.yaml

```diff
@@ -19,7 +19,8 @@ env:
   # - istio 1.5.1 envoy version
   # - latest envoy master
   ENVOY_IMAGES: >
-    envoyproxy/envoy:v1.23-latest
+    envoyproxy/envoy:v1.24-latest
```

Comment:
I think line 17-20 should be fixed two as is no longer valid. Also I think we should aim to support latest stable istio's envoy version.

Reply:
Went ahead and changed the text to target the latest Envoy releases. Perhaps coincidence, but it does seem like the previous two Envoy releases correspond to the previous two Istio 1.15 and 1.16. I think the CI is easier to reason about if we target Envoy releases directly, especially since it's often used without Istio.